src/render-background.ts

```diff
@@ -0,0 +1,48 @@
+import colorize from './colorize.js';
+import {type DOMNode} from './dom.js';
+import type Output from './output.js';
+
+const renderBackground = (
+	x: number,
+	y: number,
+	node: DOMNode,
+	output: Output,
+): void => {
+	if (node.style.backgroundColor) {
```

Comment:
Super nitpick, but would be nice to use early return here and for `if (contentWidth > 0 && contentHeight > 0)`

Reply:
I think this is unnecessary, or I can take it in a follow up PR, let's merge it